admiral.app.js

```diff
@@ -130,6 +130,11 @@ function _initializeConfig(bag, next) {
   else
     bag.config.runMode = 'production';
 
+  if (bag.env.IGNORE_TLS_ERRORS && bag.env.IGNORE_TLS_ERRORS === 'true')
```

Comment:
why will it be a string and not boolean?

Reply:
@ric03uec : currently in `envHandler.js put`, the `sed` command wraps value in double quotes. so the environment variable is loaded as a string.

https://github.com/Shippable/admiral/blob/master/common/envHandler.js#L37